ci/authn-k8s/dev/Dockerfile.inventory_base

```diff
@@ -1,7 +1,9 @@
-FROM ruby:2.3
+FROM ruby:3.0.2
 
-RUN gem install -N activesupport --version 4.2.7.1
-RUN gem install -N conjur-api:"< 5.0" conjur-cli:"< 6.0" sinatra
+RUN gem install -N activesupport --version 6.1.4.1
+RUN gem install -N conjur-api --version 5.3.6
+RUN gem install -N conjur-cli --version 6.2.5
+RUN gem install -N sinatra
```

Comment:
Why is sinatra needed explicitly?

Reply:
Not sure. Just made it more readable inside the Dockerfile. (sinatra was already there)

Reply:
This is a pretty substantial anti-pattern, but because the pattern already exists, I've added it to the list of future work.